keps/sig-node/3545-improved-multi-numa-alignment/README.md

```diff
@@ -175,13 +178,13 @@ func getDistance(sysFs sysfs.SysFs, nodeDir string) ([]uint64, error) {
 ### Implementation strategy
 
 - Introduce new flag in Kubelet called `topology-manager-policy-options`, which when specified with `prefer-closest-numa-nodes` will modify the behavior of `best-effort` and `restricted` policy to pick NUMA nodes based on average distance between them.
-- The `TopologyManagerPolicyOptions` flag is propagated to `ContainerManager` and later to `TopologyManager`.
+- The `TopologyManagerPolicyAlphaOptions` flag is propagated to `ContainerManager` and later to `TopologyManager`.
 - Enable `TopologyManager` NUMA distances discovery:
   - Temporarily add distances discovery logic into `kubelet` (similarly to [the introduction](https://github.com/kubernetes/kubernetes/commit/ecc14fe661c22f5da967a7ff50cfb3aead60905b) of `GetNUMANode()`).
   - Once `cadvisor` exposes the NUMA distance information through `MachineInfo`, remove the logic out of `kubelet` (similarly to [the removal](https://github.com/kubernetes/kubernetes/commit/a047e8aa1b705bb7e5be881fb63cf90a218b60d0) of `GetNUMANode()`).
   - The removal of the logic `kubelet` is [an Alpha to Beta graduation criteria](#alpha-to-beta-graduation).
 - When `TopologyManager` is being created it discovers distances between NUMA nodes and stores them inside `manager` struct. This is temporary until `distance` information will land into `cadvisor`.
-- Pass `TopologyManagerPolicyOptions` to best-effort and restricted policy. When this is specified best-hint is picked based on average distance between NUMA nodes. This would require modification to `compareHints` function to change how the best hint is calculated:
+- Pass `TopologyManagerPolicyAlphaOptions` to best-effort and restricted policy. When this is specified best-hint is picked based on average distance between NUMA nodes. This would require modification to `compareHints` function to change how the best hint is calculated:
```

Comment:
This should be called `TopologyManagerPolicyOptions` (corresponding to topology-manager-policy-options flag in kubelet).

Reply:
Fixed.